editor/src/components/canvas/controls/select-mode/cursor-overlay.tsx

```diff
@@ -33,8 +33,8 @@ export const CursorOverlay = React.memo(() => {
       zIndex: 9999999,
     }
     if (cursor != null) {
```

Comment:
This whole conditional can go in the bin right?

Reply:
I removed this only temporarly for experimenting with a control mouseover effect